### PR TITLE
Implement unstoppable domains

### PR DESCRIPTION
**Pull request by totop716**

### CODE REVIEW

**Comment by totop716** [comment]
[Unstoppable Domains](https://trello.com/c/JVfHW12W/152-unstoppable-domains)

File: .env [hunk 1]
```diff
@@ -32,4 +32,7 @@ REACT_APP_TRUST_WALLET_INSTALL_LINK="https://chrome.google.com/webstore/detail/t
 REACT_APP_BINANCE_CNT_URL="https://www.binancecnt.com"
 REACT_APP_BINANCE_MERCHANT_CODE="quickswap"
 REACT_APP_COINBASE_APP_ID="2eb63e45-7e6e-4699-908c-b1079d5d7436"
-REACT_APP_SPRITZ_FINANCE_LINK="https://quickswap.spritz.finance/"
\ No newline at end of file
+REACT_APP_SPRITZ_FINANCE_LINK="https://quickswap.spritz.finance/"
+REACT_APP_UNSTOPPABLE_DOMAIN_CLIENT_ID="6b358fe2-466d-4ba2-8a5e-17ef5073b053"
+REACT_APP_UNSTOPPABLE_DOMAIN_REDIRECT_URI="http://localhost:3000"
```
Comment (sameepsi) [inline .env:37]: is it correct?

Comment (totop716) [reply .env:37]: I created a new client for unstoppable domains on my side for test. We will need to create a client and replace with its info in `.env`

Comment (totop716) [reply .env]: Updated `.env` and pushed a commit

**Review by sameepsi — APPROVED** [review]

**Pull request merged**